drivers/vulkan/rendering_device_driver_vulkan.cpp

```diff
@@ -4600,6 +4697,21 @@ void RenderingDeviceDriverVulkan::command_bind_render_uniform_set(CommandBufferI
 	vkCmdBindDescriptorSets((VkCommandBuffer)p_cmd_buffer.id, VK_PIPELINE_BIND_POINT_GRAPHICS, shader_info->vk_pipeline_layout, p_set_index, 1, &usi->vk_descriptor_set, 0, nullptr);
 }
 
+void RenderingDeviceDriverVulkan::command_bind_render_uniform_sets(CommandBufferID p_cmd_buffer, VectorView<UniformSetID> p_uniform_sets, ShaderID p_shader, uint32_t p_first_set_index, uint32_t p_set_count) {
+	if (p_set_count == 0) {
+		return;
+	}
+
+	const ShaderInfo *shader_info = (const ShaderInfo *)p_shader.id;
+	VkDescriptorSet *sets = (VkDescriptorSet *)alloca(sizeof(VkDescriptorSet) * p_set_count);
```

Comment:
In the driver code we use an utility macro (not repeating the comment for every occurence):

```suggestion
	VkDescriptorSet *sets = ALLOCA_ARRAY(VkDescriptorSet, p_set_count);
```

Reply:
For some reason, we're also using the thread-local vector approach in driver code, as stated here: https://github.com/godotengine/godot/pull/99257/files#r1843284766

So, feel free to pick either. There will have to be a uniformity pass in the future.

Reply:
Done.

Switched to thread_local.